File: DEMO/DEMO/ViewController.m
//
//  ViewController.m
//  DEMO
//
//  Created by Alex D. on 3/25/16.
//  Copyright © 2016 ifnil. All rights reserved.
//

#import "ViewController.h"
#import "DEMO-swift.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet CCTabbar *tabbar;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.tabbar.tabs = @[@"3333", @"77355", @"12", @"35677772"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: DEMO/DEMO/ViewController.swift
//
//  ViewController.swift
//  DEMO
//
//  Created by Songwen on 2018/9/10.
//  Copyright © 2018 Songwen. All rights reserved.
//

import UIKit
import Network
import SystemConfiguration

class ViewController: UIViewController {

    let reach = SCNetworkReachability.reachabilityForInternetConnection
    override func viewDidLoad() {
        super.viewDidLoad()
        reach?.updateCallBack = { (r) in
            self.view.backgroundColor = (self.reach?.currentReachabilityStatus == .notReachable ? UIColor.red : UIColor.green)
        }
        reach?.start()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }
}

